css/events.css

```diff
@@ -161,6 +161,28 @@ Section Styles (background, padding, etc.)
 	margin: 0 auto;
 }
 
+.event-detail-container {
```

Comment:
Another thing to note is the header is does have a min-height of  **100vh**. this can be causing some issues with no header and needs to be reset. check out #7 for what i did